index.Rmd

```diff
@@ -20,7 +20,7 @@ Questions to be addressed:
 * Where has the band toured?
 * What are the top 5 places they have toured?
 
-# Materials and Methods
+# Materials and Methods 
```

Comment:
This is such a cool topic!!

Reply:
I like that you have multiple research questions, it's a really in depth project